README.md

```diff
@@ -104,6 +104,14 @@ Sometimes, though, you need to specify a POST or a DELETE, and you can do so wit
     # Delete rule 1
     b('rules', 1, http_method='delete')
 
+If you need to specify a keyword argument that would conflict with a Python keyword, such as `class`, simply append an underscore to it, like this:
+
+    # Set property "class" to "Hallway".
```

Comment:
Shouldnt this be "class_"?

Reply:
The name of the property being set on the bridge is “class”, but the keyword argument is “class_” to avoid clashing with the Python keyword `class`. Maybe it would be more clear if the wording were something like, “Set bridge property "class" to "Hallway"”?

Reply:
Nicely done - thanks!